### PR TITLE
featureRemoveProjectButton

### PR DESCRIPTION
**Pull request by dennis-aclue**

### CODE REVIEW

File: backend/src/main/java/de/voelkldennis/backend/projects/ProjectService.java [hunk 1]
```diff
@@ -27,5 +27,19 @@ public Optional<Project> getProjectWithId(String projectId) {
         return projectRepo.findByProjectId(projectId);
     }
 
-}
 
+    public Project deleteProjectWithId(String projectId) {
+        projectRepo.deleteByProjectId(projectId);
+        return null;
```
Comment (ChristianBorn) [inline backend/src/main/java/de/voelkldennis/backend/projects/ProjectService.java:33]: Consider returning true on successful delete-operation (Makes testing easier) or make the method a void-method

File: frontend/src/pages/projects/AddProject.tsx [hunk 2]
```diff
@@ -67,12 +67,13 @@ export default function AddProject() {
                            onChange={handleChange}
                            placeholder="project name"
                     />
+                    <p></p>
```
Comment (ChristianBorn) [inline frontend/src/pages/projects/AddProject.tsx:70]: Empty HTML-Tags should be deleted

File: frontend/src/pages/projects/ProjectCard.tsx [hunk 3]
```diff
@@ -27,29 +29,81 @@ const ProjectCard = () => {
     }, [params])
 
 
+    function deleteProject() {
+        const deleteProjectUrl = `/api/projects/${params.projectId}`
+        axios.delete(deleteProjectUrl)
+            .then((response) => {
+                setIsLoading(false)
+                console.log("Delete project with projectId SUCCESS: " + response)
```
Comment (ChristianBorn) [inline frontend/src/pages/projects/ProjectCard.tsx:37]: Production code should not have console logs

File: frontend/src/pages/projects/ProjectOverview.tsx [hunk 4]
```diff
@@ -33,13 +33,14 @@ export default function ProjectOverview() {
...
-                    <p>{projects.shortDescription.substring(0, 20)}...</p></li>
+                    <p className="projectCard__element data">{projects.projectName}</p>
+                    <p className="projectCard__element data">{projects.shortDescription.substring(0, 20)}...</p></li>
             </Link>)}</ul>
         </section>
 
-        <button onClick={() => navigate("/addProject")}>add new project</button>
+        <p>
+            <button onClick={() => navigate("/addProject")}>add new project</button>
+        </p>
```
Comment (ChristianBorn) [inline frontend/src/pages/projects/ProjectOverview.tsx:43]: p-tags are are semantically defined to display blocks of text (=paragraph)
Consider using div to wrap the button or don't wrap it at all

File: frontend/src/style/projects/addProject.css [hunk 5]
```diff
@@ -15,3 +16,19 @@
     flex-wrap: wrap;
     padding: 0;
 }
+
+input.addProjectForm__input {
+    margin: 0 10px 0 10px;
```
Comment (ChristianBorn) [inline frontend/src/style/projects/addProject.css:21]: Could be simplified:
margin: 0 10px;

**Review by ChristianBorn — APPROVED** [review]

File: frontend/src/pages/projects/AddProject.tsx (hunk 2 above)
Comment (dennis-aclue) [reply frontend/src/pages/projects/AddProject.tsx:70]: Thanks. Changed Css Margin :) and deleted <p> tag

File: frontend/src/pages/projects/ProjectOverview.tsx (hunk 4 above)
Comment (dennis-aclue) [reply frontend/src/pages/projects/ProjectOverview.tsx:43]: changed to div 👍

File: frontend/src/style/projects/addProject.css (hunk 5 above)
Comment (dennis-aclue) [reply frontend/src/style/projects/addProject.css:21]: changed. Mega!

File: backend/src/main/java/de/voelkldennis/backend/projects/ProjectService.java (hunk 1 above)
Comment (dennis-aclue) [reply backend/src/main/java/de/voelkldennis/backend/projects/ProjectService.java]: I changed the return type and the tests also.

**Review by stevenpatricklang — APPROVED** [review]

**Pull request merged**